src/java.base/share/classes/javax/crypto/Cipher.java

```diff
@@ -1783,7 +1784,7 @@ public final void init(int opmode, Certificate certificate,
     /**
      * Ensures that Cipher is in a valid state for update() and doFinal()
      * calls - should be initialized and in ENCRYPT_MODE or DECRYPT_MODE.
-     * @throws IllegalStateException if Cipher object is not in valid state.
+     * @throws IllegalStateException if Cipher object is not in valid state
```

Comment:
"Cipher" in `{@code}`? Or make it lowercase.

Reply:
Ok, I will use lowercase.